SU-HSDO/suhumsci
# HSD8-312 HSD8-504 Show external link icon and style to match

# READY FOR REVIEW

# Summary
- Show and style external link icons.

# Needed By (Date)
- 2/13

# Urgency
- low

# Steps to Test
1. checkout this branch
1. `composer install --prefer-source`
1. sync a site like archaeology `blt drupal:sync --site=archaeology --partial` (use partial to keep existing views
1. verify any external link in the main content region has an external link icon inline with the text. including heading tags.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
